runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -374,6 +375,8 @@ public int getSize() {
   private final ConcurrentMap<Long, Runnable> commitCallbacks = new ConcurrentHashMap<>();
 
   // Map of user state names to system state names.
+  // TODO(b/110224474): obsolete stateNameMap. Use transformUserNameToStateFamily in
```

Comment:
Replace this with a JIRA or a user name please.

Reply:
Done. PTAL.